azext_iot/operations/events3/_builders.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
+import uamqp
+from time import time
+
+from azext_iot.common.sas_token_auth import SasTokenAuthentication
+from azext_iot.common.utility import (parse_entity, unicode_binary_map, url_encode_str)
+
+DEBUG = True
+
+
+class AmqpBuilder():
+    def build_iothub_amqp_endpoint_from_target(self, target, duration=360):
```

Comment:
This method makes sense as a class method because there is no reliance on the instance.